update/genesis/export.go

```diff
@@ -62,15 +69,17 @@ func (se *stateExport) ExportAll(epoch uint32) error {
 		return err
 	}
 
+	defer se.writer.Finish()
+
 	err = se.exportMeta()
 	if err != nil {
 		return err
 	}
 
-	err = se.exportAllTries()
-	if err != nil {
-		return err
-	}
+	//err = se.exportAllTries()
```

Comment:
no need to comment exportAllTries function.

Reply:
you commented the code here